src/main/java/frc/robot/Robot.java

```diff
@@ -72,6 +72,10 @@ public void robotInit() {
     // report the RobotId to the SmartDashboard
     SmartDashboard.putString("RobotID", kRobotId.name());
 
+    if (isSimulation()) {
+      kRobotId = RobotId.SwerveCompetition;
+    }
+
```

Comment:
This should not be needed. Your simulation can set the RobotId preference. 

I made RobotId more abstract in a new PR.

Reply:
Just curious, how would the simulation set the RobotId preference?

Reply:
Never mind me, didn't read the comment in the code